test/io/test_ugrid.py

```diff
@@ -8,17 +8,16 @@
 import uxarray as ux
 from uxarray.constants import INT_DTYPE, INT_FILL_VALUE
 
-try:
-    import constants
-except ImportError:
-    from . import constants
+# Import centralized paths
+import sys
+sys.path.append(str(Path(__file__).parent.parent))
+from paths import *
+import constants
 
-current_path = Path(os.path.dirname(os.path.realpath(__file__)))
-
-gridfile_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30.ug"
-gridfile_RLL1deg = current_path / "meshfiles" / "ugrid" / "outRLL1deg" / "outRLL1deg.ug"
-gridfile_RLL10deg_ne4 = current_path / "meshfiles" / "ugrid" / "ov_RLL10deg_CSne4" / "ov_RLL10deg_CSne4.ug"
-gridfile_exo_ne8 = current_path / "meshfiles" / "exodus" / "outCSne8" / "outCSne8.g"
+gridfile_ne30 = OUTCSNE30_GRID
+gridfile_RLL1deg = OUTRLL1DEG_GRID
+gridfile_RLL10deg_ne4 = OV_RLL10DEG_CSNE4_GRID
+gridfile_exo_ne8 = EXODUS_OUTCSNE8
```

Comment:
Minor nitpick, but I've noticed that each testing module is redefining these variables. Shouldn't we instead use the imported variables from the `paths` module?

So for example, we'd use `OUTCSNE30_GRID` directly instead of creating a variable `gridfile_ne30`